app/controllers/garage/docs/resources_controller.rb

```diff
@@ -1,5 +1,4 @@
 require 'oauth2'
-require 'http_accept_language'
```

Comment:
It is ambiguous one `require` was removed though another was not. Are there any reason?